source/docs/articles/sites/varnish/debugging-cache.md

```diff
@@ -1,55 +1,48 @@
 ---
-title: Debugging Cache
+title: Understanding and Debugging Varnish Cache Issues
 description: Detailed information on debugging your Pantheon Drupal or WordPress Cache.
 category:
   - debugging
 keywords: varnish, cache, clear cache, caching, drupal, wordpress, cookies
 ---
-There are three ways to clear all Varnish caches for both Drupal and WordPress sites.
+In addition to clearning your CMS caches, we extend core functionality to clear all Varnish caches along our edge layer as well. This applies to Drupal and WordPress sites.
 
 ### Drupal
-The first two require you to enable the **pantheon\_api** module to allow Drupal to send the request to clear the Varnish caches.
-
-1. From Drupal: `/admin/config/devel/performance` and click **Clear all Caches**
-2. Via [Terminus](/docs/articles/local/cli/): `terminus drush --site=<site> --env=<env> cc all`
-3. From the Pantheon Dashboard: Click **Clear Caches**
-
-<div class="alert alert-info" role="alert"><h4>Note</h4>Varnish caches cannot be selectively cleared.</div>
+You must enable the **pantheon\_api** module to allow Drupal to send the request to clear the Varnish caches.
```

Comment:
Suggest linking **pantheon_api** to [What is the "pantheon_api.module"](https://pantheon.io/docs/articles/sites/code/what-is-the-pantheon_api-module/)

@sukottokun should we mention that the **pantheon_api** is not available for Drupal 8 sites? Are the methods for clearing varnish different for D8 sites?